manifests/charts/istio-control/istio-discovery/files/gateway-injection-template.yaml

```
@@ -13,9 +13,13 @@ metadata:
   }
 spec:
   securityContext:
+  {{- if .Values.gateways.securityContext }}
```

```suggestion
  {{- if .Values.gateways.securityContext }}
```

since we aren't updating all the gateway charts to respect this, should this be `.Values.managedGateways.securityContext`? Or `.Values.injectedGateways.securityContext`?


(I know you asked for suggestions earlier @nicole-lihui - apologies for not responding sooner)

---

@bleggett  I'm just happy to have you response, I don't mind the time 🫶 

I've alse tried adding a similar field like `injectedGateways`. and then I realized that when users need to customize the ``securityContext`, it implies that their enviroment has unique requirements.  
In such case, users would likely need to comtomize this across the board.

Additionally, `injectedGateways` is a new concept for users(who are not istio developers but simply end-users), and I'm weighing the cost of introducing it.  And then I just want to provide a simply way to customize the securityContext.

 💭 🤔   actually, I'm not even sure if my thinking is correct .....

---

Yep it's just that if I was looking at this, I might reasonably assume `Values.gateways.securityContext` applies to all `gateways` and not just the injected ones.

I agree either way has issues. It's a nit. I'm ok to leave it as is.